src/main/java/com/searchcode/app/jobs/IndexGitRepoJob.java

```diff
@@ -363,18 +364,21 @@ public RepositoryChanged cloneGitRepository(String repoName, String repoRemoteLo
         Singleton.getLogger().info("Attempting to clone " + repoRemoteLocation);
 
         try {
-            CloneCommand cloneCommand = Git.cloneRepository();
-            cloneCommand.setURI(repoRemoteLocation);
-            cloneCommand.setDirectory(new File(repoLocations + "/" + repoName + "/"));
-            cloneCommand.setCloneAllBranches(true);
-            cloneCommand.setBranch(branch);
-
-            if(useCredentials) {
-                cloneCommand.setCredentialsProvider(new UsernamePasswordCredentialsProvider(repoUserName, repoPassword));
+            Git g = null;
+            try
+            {
+                CloneCommand cloneCommand = Git.cloneRepository();
+                cloneCommand.setURI(repoRemoteLocation);
+                cloneCommand.setDirectory(new File(repoLocations + "/" + repoName + "/"));
+                cloneCommand.setCloneAllBranches(true);
+                cloneCommand.setBranch(branch);
+                if(useCredentials)
```

Comment:
Can we surround this with { } please? It just makes things easier. I will add a style guide to avoid this problem in the future.